chrome/content/zotero/preferences/quickCopySiteEditor.xul

```diff
@@ -46,6 +47,7 @@
 				var io = window.arguments[0];
 				io.domain = document.getElementById('zotero-quickCopy-domain').value;
 				io.format = document.getElementById('zotero-quickCopy-menu').value;
+				io.locale = document.getElementById('locale-menu').label !== '' ? document.getElementById('locale-menu').value : '';
```

Comment:
This got changed/improved when I added locale list disabling for styles with default-locales: https://github.com/rmzelle/zotero/commit/d6da32888caa0576878e05360817e6e112db5867